benchmarks/blake.cc

```diff
@@ -288,28 +344,6 @@ HACL_blake2b_32_streaming(benchmark::State& state)
 
 BENCHMARK(HACL_blake2b_32_streaming)->Setup(DoSetup);
 
-static void
```

Comment:
Is there a reason to drop these two?
@karthikbhargavan looks like you did this change.